ci-scripts/linux/tahoma-build.sh

```diff
@@ -21,9 +16,9 @@ then
 fi
 
 export LD_LIBRARY_PATH=/usr/local/lib:$LD_LIBRARY_PATH
-cmake ../sources  $CANON_FLAG \
+CC="gcc" cmake ../sources  $CANON_FLAG \
```

Comment:
As this is setting up to build T2D, this change also impacts the main build, not just libTIFF build.

Need to see if T2D build is really using ccache or not. Hard to tell from the Github logs. if so, this change would disable that causing build times to increase.